### PR TITLE
Python 3 support for urlparse4

### PR DESCRIPTION
**Pull request by malloxpb**
Hey guys, I made a python 3 support for this project. Can you take a look at it when you have a chance? 😃 #5

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -83,6 +83,5 @@
         "Topic :: Software Development :: Libraries"
     ],
     long_description=long_description,
-    ext_modules=ext_modules,
-    include_package_data=True
```
Comment (lopuhin) [inline setup.py]: is ``include_package_data=True`` removed on purpose?

Comment (malloxpb) [reply setup.py]: I remember I got some trouble with it at the beginning but now it's resolved. Let me put it back real quick!

File: urlparse4/__init__.py [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # https://github.com/python/cpython/blob/2.7/Lib/urlparse.py
 
-from urlparse import *
+import six
+
+from six.moves.urllib.parse import urlparse, urlsplit, urljoin
```
Comment (lopuhin) [inline urlparse4/__init__.py]: I think the idea behind ``from urlparse import *`` was to provide all functions from urlparse module, and then override some of them below, so this should be ``from six.moves.urllib.parse import *`` now

File: urlparse4/__init__.py [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
+
 
 _original_urlsplit = urlsplit
 _original_urljoin = urljoin
 
-from cgurl import urlsplit, urljoin
+from cgurl import url_split, url_join
```
Comment (lopuhin) [inline urlparse4/__init__.py]: this functions should override urlsplit, urljoin from urlparse, so names should stay the same

Comment (malloxpb) [reply urlparse4/__init__.py]: I had some ImportError before so I tried to change it as an experiment. I will put it back now 😄

File: urlparse4/__init__.py (hunk 2 above)
Comment (malloxpb) [reply urlparse4/__init__.py]: I got this error when importing all the modules of urllib:
```
NameError: name 'urlsplit' is not defined
```

Comment (malloxpb) [reply urlparse4/__init__.py]: Should I import all the functions in urllib manually?

**Review by lopuhin — APPROVED** [review]
Looks good to me!

**Pull request closed without merging**